### PR TITLE
exported _http_server and forcibly starting

### PR DESCRIPTION
**Pull request by kzidane**
For serve in file browser's context menu.

### CODE REVIEW

File: files/etc/profile.d/ide50.sh [hunk 1]
```diff
@@ -156,6 +156,11 @@ flask()
 # http-server
 _http_server()
 {
+
+    # kill running instances of http-server before starting new one
+    pid=$(pgrep -f http-server)
```
Comment (dmalan) [inline files/etc/profile.d/ide50.sh]: Hm, can we kill 8081 only? Since 8080 is meant to be used by students at command line.

**Comment by kzidane** [comment]
@dmalan had to separate to a script because the subshell started by the plugin doesn't see the `_http_server` function and we can't export it since `ide50.sh` has to be `sh`-compatible. Should we do the same for `flask` wrapper for consistency?

File: files/etc/profile.d/ide50.sh (hunk 1 above)
Comment (kzidane) [reply files/etc/profile.d/ide50.sh]: Not sure I follow. Do you want to kill anything listening on 8081 and listen on that port when right- (or Ctrl-) clicking on a directory and choosing **Serve**?

**Comment by kzidane** [comment]
@dmalan any takes on that one too? Needed for the **Serve** menu item.

**Pull request merged**